modules/common/kv_db/kv_db.cc

```diff
@@ -95,6 +96,7 @@ std::string KVDB::Get(const std::string &key,
 
   std::string value;
   const auto status = GetDB()->Get(options, key, &value);
+  CHECK(status.ok() || status.IsNotFound()) << status.ToString();
```

Comment:
I prefer "AERROR_IF(status.ok() || !default_value.empty())".
Only show error if the key doesn't exist and no default value is set. Even this case is sometimes the expected behavior.
We could add another API named "GetOrDie(key)" to do CHECK to guarantee existence.